types/window.d.ts

```diff
@@ -2,6 +2,10 @@ interface IntegrationHelpers {
   updateAccountAddress(address: string): void;
   findMarketId(marketDescription: string): void;
   hasDisclaimerModalBeenDismissed(): boolean;
+  logout(): void;
+  getAccountData(): void;
+  formatRep(value: string): void;
+  formatEth(value: string): void;
 }
```

Comment:
shouldn't `formatRep` and `formatEth` return object?
better would be a formatted object